engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-traces-modal.controller.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostsBulkTracesController
+ *
+ * @requires $scope
+ * @requires $uibModalInstance
+ * @requires HostBulkAction
+ * @requires Notification
+ * @requires Nutupane
+ * @requires BastionConfig
+ * @requires hostIds
+ * @requires HostTracesResolve
+ * @requires translate
+ *
+ * @description
+ *   Provides the functionality to support resolving traces on multiple hosts.
+ */
+/*jshint camelcase:false*/
+angular.module('Bastion.content-hosts').controller('ContentHostsBulkTracesController',
+    ['$scope', '$uibModalInstance', 'HostBulkAction', 'Notification', 'Nutupane', 'BastionConfig', 'hostIds', 'HostTracesResolve', 'translate',
+    function ($scope, $uibModalInstance, HostBulkAction, Notification, Nutupane, BastionConfig, hostIds, HostTracesResolve, translate) {
+
+        var tracesNutupane = new Nutupane(HostBulkAction, hostIds, 'traces');
+        tracesNutupane.enableSelectAllResults();
+        tracesNutupane.masterOnly = true;
+        $scope.table = tracesNutupane.table;
+        $scope.remoteExecutionPresent = BastionConfig.remoteExecutionPresent;
+
+        $scope.performViaRemoteExecution = function() {
+            var traceids = _.map($scope.table.getSelected(), 'id');
+
+            var onSuccess = function () {
+                var message = translate('Successfully initiated restart of services.');
+                Notification.setSuccessMessage(message, {
+                    link: {
+                        children: translate("Click here to view the jobs."),
```

Comment:
We typically refrain from saying 'click here' since our links are obvious. I vote for rewording this: `View job invocations.`

Reply:
Fixed